scanner/src/main/kotlin/ScanResultsStorage.kt

```diff
@@ -144,20 +149,28 @@ abstract class ScanResultsStorage {
                 maxPoolSize = LocalScanner.NUM_STORAGE_THREADS + 3
             )
 
+            log.info {
+                "Using Postgres storage with URL '${config.url}' and schema '${config.schema}'."
+            }
+
             return PostgresStorage(dataSource)
         }
 
         /**
          * Create a [ClearlyDefinedStorage] based on the [config] passed in.
          */
         private fun createClearlyDefinedStorage(config: ClearlyDefinedStorageConfiguration): ScanResultsStorage =
-            ClearlyDefinedStorage(config)
+            ClearlyDefinedStorage(config).also {
+                log.info { "Using ClearlyDefined storage with URL '${config.serverUrl}'." }
+            }
 
         /**
          * Configure a [Sw360Storage] as the current storage backend.
          */
         private fun configureSw360Storage(config: Sw360StorageConfiguration): ScanResultsStorage =
-            Sw360Storage(config)
+            Sw360Storage(config).also {
```

Comment:
While touching here, mind adding another commit that renames `configureSw360Storage` to `createSw360Storage` for consistency?